dsperse/src/compile/utils/compiler_utils.py

```
@@ -217,17 +266,16 @@ def build_model_level_ezkl(payload_rel: Dict[str, Optional[str]], calibration_re
                 model_level_ezkl[k] = v
         return model_level_ezkl
 
-
     @staticmethod
-    def update_slice_metadata(idx: int, filepath: str | Path, success: bool, file_paths: Dict[str, str | None], backend_name: str = "ezkl"):
+    def update_slice_metadata(idx: int, filepath: str | Path, success: bool, compilation_info: Dict[str, Any], backend_name: str = "ezkl"):
```

Ruff (0.14.11)

Unused static method argument: `success`

(ARG004)